draft-lenders-dns-over-coap.md

```diff
@@ -265,6 +265,17 @@ the server.
 CoAP/CoRE Integration
 =====================
 
+DoC server considerations
+-------------------------
+To reduce the number of message exchanges within an LLN, a DoC server MAY try
+to resolve CNAME chains received in DNS responses from an upstream DNS resolver.
```

Comment:
RFC 1034 (e.g., \S 5.2.2 and \S 5.3.3) and RFC 1035 (e.g., \S 7.2) make clear that a CNAME chain should be resolved until the original query is answered. This means if a client requests `abc.net A` and the record is `abc.net CNAME xyz.net; xyz.net A 10.20.30.40`, the client sends a single query and the resolver a single reply.

rewording:

In the case of CNAME records, a DoC server SHOULD follow common DNS resolver behavior {{?RFC1034}} by resolving a CNAME until the originally requested resource record is reached. This reduces the number of message exchanges within an LLN.

Reply:
Done